docs/release-notes/NuGet-5.9.md

```diff
@@ -0,0 +1,252 @@
+
+* Build of NuGet libraries is broken when assembly signing is disabled  - [#10173](https://github.com/NuGet/Home/issues/10173)
+
+* [Test Failure][Localization] The information from command ‘get-help NuGet’ in PMC is not localized - [#4604](https://github.com/NuGet/Home/issues/4604)
+
+* Design for warning on improper casing for PackageLicenseFile and PackageIcon - [#10049](https://github.com/NuGet/Home/pull/10049)
+
+* No error occurred after packing and installing a package with an invalid .txt format icon - [#9311](https://github.com/NuGet/Home/issues/9311)
+
+* Remove Temp fix on patching - [#8952](https://github.com/NuGet/Home/issues/8952)
+
+* Cannot run Nuget.Client library - [#10390](https://github.com/NuGet/Home/issues/10390)
+
+* Clean-up the contributing docs - [#10399](https://github.com/NuGet/Home/issues/10399)
+
+* Enhance the coding guidelines - add rule for parameter names & namespace names - [#10485](https://github.com/NuGet/Home/issues/10485)
+
+* tfm_raw error happening for some TFMs during pack - [#10656](https://github.com/NuGet/Home/issues/10656)
+
+**StillOpens:**
+
+* PM UI:  not automatically closed when project unloaded/removed - [#9935](https://github.com/NuGet/Home/issues/9935)
+
+* E2E test GetPackageUpdatesAfterSwitchToSourceThatDoesNotContainInstalledPackageId is flaky - [#10254](https://github.com/NuGet/Home/issues/10254)
+
+* Race condition during restore for DotnetCliToolReference packages in the default parallel processing mode - [#10075](https://github.com/NuGet/Home/issues/10075)
+
+* Test:  TestPackageManager_UpgradePackageFor_TopParentProject_Success is flaky - [#10208](https://github.com/NuGet/Home/issues/10208)
+
+* Test: TestPacMan_PreviewInstallPackage_BuildIntegrated_RaiseTelemetryEventsWithWarningCode is flaky - [#10093](https://github.com/NuGet/Home/issues/10093)
+
+* Telemetry: Replace EmitTelemetryEvent with proper telemetry activities - [#9581](https://github.com/NuGet/Home/issues/9581)
+
+**[List of all issues fixed in this release - 5.9](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=5f6be8c10485c0236b7ef889)**
+
+**[List of commits in this release - 5.8.1](https://github.com/NuGet/NuGet.Client/compare/5.8.1.7021...5.9.0.7134)**
+
+### Community contributions
+
+Thank you to all the contributors who helped make this NuGet release awesome!
+
+|Who|PRs|Issues|
+|----|----|----|
+[omajid](https://github.com/omajid) | [3865](https://github.com/NuGet/NuGet.Client/pull/3865) | Copy-paste error in V2FeedPackageInfo - [#10480](https://github.com/NuGet/Home/issues/10480)
+[marcin-krystianc](https://github.com/marcin-krystianc) | [3812](https://github.com/NuGet/NuGet.Client/pull/3812) | Missing tests for the case where packages are referenced with PrivateAssets="All" attribute - [#10397](https://github.com/NuGet/Home/issues/10397)
```

Comment:
Some of the issues in here are not called out above. That seems inconsistent.

Reply:
This one has "Type:Test" label, so tool wouldn't pickup. I searched all community contributions manually  and added here. Do we recognize and mention community contribution to `Tests` without actual code change contribution? If not then we can remove this.

Reply:
I'm fine with leaving in that case. 

I think I'd normally lean toward excluding these, but I don't want to make that decision unilaterally :) 

cc @NuGet/nuget-client

Reply:
Let's circle back to this if we really need it after release. I'm hopeful to get notes out today & thus going to merge this PR soon.